### PR TITLE
Include account balances for smaller dimensions

### PR DESCRIPTION
**Pull request by benzap**
- fixes issue #344

# Preview
a

### CODE REVIEW

File: src/memefactory/styles/app_menu.clj [hunk 1]
```diff
@@ -54,9 +66,74 @@
         :line-height (em 1.4)
         :color (color :menu-logo)}]]
 
+     [:.accounts
```
Comment (madvas) [inline src/memefactory/styles/app_menu.clj]: Why to copy paste whole styling from app_bar.cljs? Cannot this be styled just once and reused as a component?

File: src/memefactory/ui/components/app_layout.cljs [hunk 2]
```diff
@@ -141,24 +141,44 @@
...
-                         text]]
-                       (when children
-                         [app-menu children active-page (inc depth)])])
-                    items))])))
+     [:<>
+      ;; Account Balance Header for Mobile
+      (when (and active-account (= depth 0))
+        [:div.accounts
+         [:div.dank-section
+          [:div.dank-logo
```
Comment (madvas) [inline src/memefactory/ui/components/app_layout.cljs]: Please make logos defined in CSS. This change goes for app-bar as well. https://github.com/district0x/memefactory/blob/eafabdc0b82d4bd0b970a0d5ee63ea609c29ca64/src/memefactory/ui/components/app_layout.cljs#L108

Comment (madvas) [reply src/memefactory/ui/components/app_layout.cljs]: Also, good would be if you could make the balance section into reagent component. So it's reused just as 1 line.

File: src/memefactory/styles/app_menu.clj (hunk 1 above)
Comment (benzap) [reply src/memefactory/styles/app_menu.clj]: I more wanted to post a preview to see if it's the correct styling. I can do further work in refactoring it and making it less DRY.

**Pull request merged**